### ISSUE TITLE
[DataCap Application] Smithsonian Institution

### ISSUE THREAD

**Issue by lijo76618** [filplus-bookkeeping/IPFSCN#31]
### Version

1

### DataCap Applicant

Smithsonian Institution

### Project ID

01

### Data Owner Name

Smithsonian Institution

### Data Owner Country/Region

United States

### Data Owner Industry

Web3 / Crypto

### Website

https://www.si.edu/visit

### Social Media Handle

https://www.si.edu/visit

### Social Media Type

Slack

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

8PiB

### Expected size of single dataset (one copy)

1PiB

### Number of replicas to store

8

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f1j7vbm3p2qadwjn3yupi2bk4sye3zucwqwahnu2y

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
The Smithsonian’s mission is the "increase and diffusion of knowledge" and has been collecting since 1846. The Smithsonian, through its efforts to digitize its multidisciplinary collections, has created millions of digital assets and related metadata describing the collection objects. On February 25th, 2020, the Smithsonian released over 2.8 million CC0 interdisciplinary 2-D and 3-D images, related metadata, and additionally, research data from researches across the Smithsonian. The 2.8 million "open access" collections are a subset of the Smithsonian’s 155 million objects, 2.1 million library volumes and 156,000 cubic feet of archival collections held in 19 museums, 9 research centers, libraries, archives and the National Zoo. Digitization of collections is ongoing.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
The Smithsonian’s mission is the "increase and diffusion of knowledge" and has been collecting since 1846. The Smithsonian, through its efforts to digitize its multidisciplinary collections, has created millions of digital assets and related metadata describing the collection objects. On February 25th, 2020, the Smithsonian released over 2.8 million CC0 interdisciplinary 2-D and 3-D images, related metadata, and additionally, research data from researches across the Smithsonian. The 2.8 million "open access" collections are a subset of the Smithsonian’s 155 million objects, 2.1 million library volumes and 156,000 cubic feet of archival collections held in 19 museums, 9 research centers, libraries, archives and the National Zoo. Digitization of collections is ongoing.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

None

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
After we download data from the Internet, the data is cut into disks through Singularity, and then the hard disk is mailed to the SPs.
```

### Please share a sample of the data

```text
s3://smithsonian-open-access/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Filmine

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f02852273 London
f02984331 Singapore
f02883857 Singapore
f02973061 Russia
f02889193 Vietnam
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client, Singularity

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by ipfscn** [comment]
You mentioned that the data will be cut by the Singularity tool and then mailed to Storage Providers (SPs). How do you ensure the security and integrity of the data during the mailing process? Additionally, how do you plan to handle and store this data to ensure it meets the requirements of the Filecoin network?

**Comment by lijo76618** [comment]
We have developed our own data processing system based on Singularity. After processing the data, we transfer it to hard drives in batches. Each hard drive will be labeled to ensure that data is not lost, duplicated, or mixed up, and to ensure that all processes comply with FIL+ requirements.

**Comment by ipfscn** [comment]
Do all SPs support Spark? Are you ready to start sealing?

**Comment by lijo76618** [comment]
Dear, thank you for your question. We are ready. SP supports spark retrieval. We hope to get your help.

**Comment by lijo76618** [comment]
checker:manualTrigger

**Comment by lijo76618** [comment]
I hope to continue to receive support, thank you

**Comment by ipfscn** [comment]
checker:manualTrigger

**Comment by ipfscn** [comment]
checker:manualTrigger

**Comment by ipfscn** [comment]
Looks normal

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by ipfscn** [comment]
Can you explain why some new SPs have been added this time?
It seems that these SPs are using VPN, and according to our rules, we do not accept SPs that use VPN
So, we can no longer support you.
I hope you can shut them down.

**Comment by lijo76618** [comment]
Hello, sorry for not explaining the situation to you in advance:
f02852273 London
f02984331 Singapore
f02883857 Singapore
f02973061 Russia
f02889193 Vietnam
These five are our customers' SPs, and they are also the SPs we disclosed when filling out the application form
After you signed 1P for us, we added a new customer, and this customer's SP used VPN. Because they were once attacked by hackers and lost a total of 8593 FIL, from then on, all their SPs will use VPN.
But it seems that we should not help them, because they have too many SPs and it looks messy, which makes this LDN look strange and even affects your trust in us, which may cause us to no longer be able to get your help.
If we no longer cooperate with this customer from now on, are you willing to continue to support us?
